### PR TITLE
Catch and swallow no-node exceptions (#93)

### PR DESCRIPTION
**Pull request by Crim**
Handle race condition between checking if a node exists, and removing the node.

Backporting bugfix to 0.9.x release: #92 

Cuts bugfix version 0.9.2

### CODE REVIEW

**Comment by Crim** [comment]
@stanlemon backporting this to 0.9.x branch

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [PR-86](https://github.com/salesforce/storm-dynamic-spout/pull/86) Sideline payload null checking
 - [PR-88](https://github.com/salesforce/storm-dynamic-spout/pull/88) Sideline payload null checking
 
+## 0.9.2 (2017-12-13)
```
Comment (stanlemon) [inline CHANGELOG.md]: You’ll need to do a separate commit to master with the change log after this FYI.

Comment (Crim) [reply CHANGELOG.md]: Going to just merge that change straight into master.
